api/models/site.js

```diff
@@ -80,6 +116,9 @@ module.exports = (sequelize, DataTypes) => {
     },
     demoDomain: {
       type: DataTypes.STRING,
+      validate: {
+        isEmptyOrUrl,
```

Comment:
The `validate` options added here prevent Site models from being created or updated with invalid URLs.